compose-up.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+cd $(dirname $0)
+set -e
+
+kind_state=$(docker container inspect hf-kind -f '{{ .State.Status }}')
```

Comment:
small change needed if we go with `k3d` instead of `kind`, see https://github.com/hobbyfarm/hobbyfarm/issues/78#issuecomment-597318058